setup.cfg

```diff
@@ -34,8 +34,8 @@ python_requires = >=3.7
 install_requires =
     scikit-learn>=1.0
     nflows>=0.14
-    pandas>=1.3,<2.0
-    torch>=1.10,<2.0
+    pandas>=2.0
```

Comment:
Please don't change the versioning for torch and pandas for now, there are other components affected.

Reply:
I have updated